project/source/Scripts/UI/Camera.gd

```diff
@@ -44,7 +44,7 @@ func GetAllowedRect() -> Rect2:
 	for labobject in get_tree().get_nodes_in_group("LabObjects"):
```

Comment:
Your commits look good! This is the only thing I can find that isn't snake case.